### PR TITLE
Small CSS changes and tried backend, but not done yet

### PR DESCRIPTION
**Pull request by a5h101**

### CODE REVIEW

File: Rentit-frontend/src/pages/login/login.jsx [hunk 1]
```diff
@@ -1,84 +1,77 @@
...
+          <div className='line lineLogin'></div>
+        </div>
+        <div className='wlcmTxt'>
+          <h3> Choose any Login method</h3>
+        </div>
+        <Animation />
+        <div className='signIn'>
+          <LoginSocialGoogle
+            client_id={
+              '980638331446-9io0sa1et63bgfsqu2n0282dbag0062g.apps.googleusercontent.com'
```
Comment (aditya-190) [inline Rentit-frontend/src/pages/login/login.jsx:39]: Please add these values to your .env files.

File: Rentit-frontend/src/pages/login/login.jsx [hunk 2]
```diff
@@ -1,84 +1,77 @@
...
+            }
+            scope='openid profile email'
+            discoveryDocs='claims_supported'
+            access_type='offline'
+            onResolve={({ provider, data }) => {
+              console.log(provider, data);
```
Comment (aditya-190) [inline Rentit-frontend/src/pages/login/login.jsx:45]: Also remove all the consoles before merging the changes.

File: Rentit-frontend/src/pages/profile/profile.jsx [hunk 3]
```diff
@@ -13,6 +14,7 @@ const Profile = () => {
 
   return (
     <>
+      <Subnavbar />
```
Comment (aditya-190) [inline Rentit-frontend/src/pages/profile/profile.jsx:17]: Pls Follow the General Naming convection's throughout your web app. For more details, check this out -> https://climbtheladder.com/10-react-naming-conventions-best-practices/

File: Rentit-frontend/src/pages/testPage/test.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// const Test = () => {
```
Comment (aditya-190) [inline Rentit-frontend/src/pages/testPage/test.jsx:1]: If you are not using this component anymore, this can be removed.

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Please make required changes before merging the changes.

File: Rentit-frontend/src/pages/login/login.jsx (hunk 1 above)
Comment (a5h101) [reply Rentit-frontend/src/pages/login/login.jsx:39]: Ok

File: Rentit-frontend/src/pages/login/login.jsx (hunk 2 above)
Comment (a5h101) [reply Rentit-frontend/src/pages/login/login.jsx:45]: ok

**Comment by aditya-190** [comment]
Hey you should not merge the changes until it is approved. You have to make the changes and re-request to review it again.

**Pull request merged**